src/formatter/summary_formatter.js

```diff
@@ -14,9 +14,11 @@ export default class SummaryFormatter extends Formatter {
   }
 
   isTestCaseWarning(testCase) {
-    return _.includes(
-      [Status.PENDING, Status.UNDEFINED],
-      testCase.result.status
+    return (
+      _.includes(
+        [Status.RETRY, Status.PENDING, Status.UNDEFINED],
+        testCase.result.status
+      ) || testCase.attemptNumber > 1
```

Comment:
Do we need to check the attempt number? Seems like that would show later passed attempts. Can we have a test for the changes here?

Reply:
I will review this, since I added a check for the `RETRY` status, testing the attempt number may just be a left over.